torchx/schedulers/__init__.py

```diff
@@ -22,20 +22,38 @@ def __call__(self, session_name: str, **kwargs: object) -> Scheduler:
         ...
 
 
+def try_get_ray_scheduler():  # pyre-ignore[3]
```

Comment:
do you need here both import try-catch and has-ray function? Can we remove has_ray?